charts/alfresco-share/templates/ingress.yaml

```diff
@@ -6,8 +6,7 @@ We only support nginx ingress for now: https://alfresco.atlassian.net/browse/OPS
 */}}
 {{- if not (semverCompare ">=1.18-0" .Capabilities.KubeVersion.GitVersion) }}
   {{- $_ := unset .Values.ingress.annotations "kubernetes.io/ingress.class" }}
-  {{- $_ := set .Values.ingress.annotations "kubernetes.io/ingress.class" "nginx" }}
-*/}}
+  {{- $_ = set .Values.ingress.annotations "kubernetes.io/ingress.class" "nginx" }}
```

Comment:
`:=`?

Reply:
Once the var has been defined and assigned using `:=` you can update it with a regular `=` assignment.
(Though both seem to work)

Reply:
source: https://go.dev/doc/go1.11#minor_library_changes (text/template)